ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -18,6 +18,9 @@ export const BasicInformation = ({ workspace, activeStatus }: Props) => (
       <WorkspaceInfoField labelText='Active Status'>
         {activeStatus}
       </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Billing Status'>
```

Comment:
Unrelated, but would have helped debug this user's problem sooner.

Reply:
can u explain a bit more why this would've helped?

Reply:
inactive billing causes many problems.  This field is easy to miss because it's in the middle
<img width="515" alt="Screenshot 2024-03-21 at 10 56 27 AM" src="https://github.com/all-of-us/workbench/assets/2701406/bac311e6-b4a9-43cb-9fbb-4195b6d50c3e">

So I'm thinking it would work better at the top, along with the other status